docs/dev/contributing.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to the App
+
+The project is packaged with a light [development environment](dev_environment.md) based on `docker-compose` to help with the local development of the project and to run tests.
+
+The project is following Network to Code software development guidelines and is leveraging the following:
+
+- Python linting and formatting: `black`, `pylint`, `bandit`, `flake8`, and `pydocstyle`.
+- YAML linting is done with `yamllint`.
+- Django unit test to ensure the plugin is working properly.
+
+Documentation is built using [mkdocs](https://www.mkdocs.org/). The [Docker based development environment](dev_environment.md#docker-development-environment) automatically starts a container hosting a live version of the documentation website on [http://localhost:8001](http://localhost:8001) that auto-refreshes when you make any changes to your local files.
+
+## Branching Policy
+
+!!! warning "Developer Note - Remove Me!"
+    What branching policy is used for this project and where contributions should be made.
+
+## Release Policy
+
+!!! warning "Developer Note - Remove Me!"
+    How new versions are released.
```

Comment:
Proposing the following Release Policy:

```suggestion
Nautobot Capacity Metrics currently has no intended release schedule, and will release new features in minor versions.

When a new release of any kind (e.g. from `develop` to `main`, or a release of a `stable-<major>.<minor>`) is created the following should happen.

- A release PR is created:
    - Add and/or update to the changelog in `docs/admin/release_notes/version_<major>.<minor>.md` file to reflect the changes.
    - Update the mkdocs.yml file to include updates when adding a new release_notes version file.
    - Change the version from `<major>.<minor>.<patch>-beta` to `<major>.<minor>.<patch>` in `pyproject.toml`.
    - Set the PR to the proper branch, e.g. either `main` or `stable-<major>.<minor>`.
- Ensure the tests for the PR pass.
- Merge the PR.
- Create a new tag:
    - The tag should be in the form of `v<major>.<minor>.<patch>`.
    - The title should be in the form of `v<major>.<minor>.<patch>`.
    - The description should be the changes that were added to the `version_<major>.<minor>.md` document.
- If merged into `main`, then push from `main` to `develop`, in order to retain the merge commit created when the PR was merged.
- If there is a new `<major>.<minor>`, create a `stable-<major>.<minor>` for the **previous** version, so that security updates to old versions may be applied more easily.
- A post release PR is created:
    - Change the version from `<major>.<minor>.<patch>` to `<major>.<minor>.<patch + 1>-beta` in `pyproject.toml`.
    - Set the PR to the proper branch, e.g. either `develop` or `stable-<major>.<minor>`.
    - Once tests pass, merge.

```